integration-libs/s4om/src/schedule-lines/schedule-lines.component.html

```diff
@@ -0,0 +1,44 @@
+<ng-container *ngIf="orderEntry$ | async as orderEntry">
+  <ng-container *ngIf="hasScheduleLines(orderEntry)">
+    <div>
```

Comment:
for what is this `div` element for? Are you able to remove it? (if styling doesn't require it)

Reply:
this is not necessary and can be removed